### ISSUE TITLE
What is TimerAppComponent?

### ISSUE THREAD

**Issue by chrillewoodz** [douglasduteil/angular-batscanner#20]
It's unclear what `TimerAppComponent` is, is that the root app component or what?

**Comment by thematan** [comment]
Yes you should replace it with your root app component (default is usually AppComponent)
